### PR TITLE
Format update

### PR DESCRIPTION
**Pull request by WilcoFiers**
Update existing rules to the new format

### CODE REVIEW

File: _rules/SC1-1-1-aria-describedby.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC1-1-1-aria-describedby
 name: Proper use of aria-describedby
 test_mode: semi-automatic
+environment: Web Browser
 
 criteria:
```
Comment (annethyme) [inline _rules/SC1-1-1-aria-describedby.md]: success_criterion:

Comment (EmmaJP) [inline _rules/SC1-1-1-aria-describedby.md]: Should this be "success_criterion:" ?

File: _rules/SC1-1-1-css-image.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC1-1-1-css-image
 name: CSS background decorative
 test_mode: semi-automatic
+environment: Web Browser
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC1-1-1-css-image.md]: success_criterion:

File: _rules/SC1-1-1-longdesc.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC1-1-1-longdesc
 name: Proper use of longdesc
 test_mode: semi-automatic
+environment: Web Browser
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC1-1-1-longdesc.md]: success_criterion:

File: _rules/SC1-4-2-audio-control-audio.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC1-4-2-audio-control-audio
 name: Audio control
 test_mode: semi-automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC1-4-2-audio-control-audio.md]: success_criterion:

File: _rules/SC2-4-7-focus-in-viewport.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC2-4-7-focus-in-viewport
 name: Focused element visible in viewport
 test_mode: automatic
+environment: WebDriver
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC2-4-7-focus-in-viewport.md]: success_criterion:

File: _rules/SC3-1-1-html.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-1-html
 name: Primary language of page
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-1-html.md]: success_criterion:

File: _rules/SC3-1-1-text.md [hunk 7]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-1-text
 name: HTML lang matches text
 test_mode: semi-automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-1-text.md]: success_criterion:

File: _rules/SC3-1-1-xml-lang.md [hunk 8]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-1-xml-lang
 name: Use identical xml:lang and lang attributes
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-1-xml-lang.md]: success_criterion:

File: _rules/SC3-1-2-lang.md [hunk 9]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-2-lang
 name: Languages within the body
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-2-lang.md]: success_criterion:

File: _rules/SC3-1-2-text.md [hunk 10]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-2-text
 name: Lang attribute matches text
 test_mode: semi-automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-2-text.md]: success_criterion:

File: _rules/SC3-1-2-xml-lang.md [hunk 11]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC3-1-2-xml-lang
 name: Language of Parts
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC3-1-2-xml-lang.md]: success_criterion:

File: _rules/SC4-1-1-accesskey.md [hunk 12]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC4-1-1-accesskey
 name: Provide unique accesskeys
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC4-1-1-accesskey.md]: success_criterion:

File: _rules/SC4-1-1-id.md [hunk 13]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC4-1-1-id
 name: Define ids for elements
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC4-1-1-id.md]: success_criterion:

File: _rules/SC4-1-1-idref.md [hunk 14]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC4-1-1-idref
 name: Reference elements
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC4-1-1-idref.md]: success_criterion:

File: _rules/SC4-1-1-idrefs.md [hunk 15]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC4-1-1-idrefs
 name: Reference multiple elements
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC4-1-1-idrefs.md]: success_criterion:

File: _rules/SC4-1-1-tag-nesting.md [hunk 16]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC4-1-1-tag-nesting
 name: Tags nested correctly
 test_mode: automatic
+environment: Markup Document
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC4-1-1-tag-nesting.md]: success_criterion:

File: _rules/SC2-2-1+SC3-2-5-meta-refresh.md [hunk 17]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC2-2-1+SC3-2-5-meta-refresh
 name: Meta refresh and redirect is not used
 test_mode: automatic
+environment: DOM Structure
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC2-2-1+SC3-2-5-meta-refresh.md]: success_criterion:

File: _rules/SC1-1-1-text-alternative.md [hunk 18]
```diff
@@ -2,6 +2,7 @@
 rule_id: SC1-1-1-text-alternative
 name: Provision of short text alternative
 test_mode: semi-automatic
+environment: Web Browser
 
 criteria:
```
Comment (2biazdk) [inline _rules/SC1-1-1-text-alternative.md]: success_criterion:

File: _rules/SC1-1-1-aria-describedby.md [hunk 19]
```diff
@@ -29,69 +30,34 @@ This test checks the sufficient provision of a long text description for element
...
-| Testcase | {{ page.rule_id }}
-| ID       | {{ page.rule_id }}-fail2
-| Error    | None of the elements referenced by aria-describedby exists.
+else, return [step2-fail](#step2-fail)
 
 ### Step 3
 
-Test mode: [manual][MANUAL]
-
 Concatenate the results of [Text Alternative Computation][TXTALT] Algorithm run on the element itself and assign it to variable T1 and on all elements referenced by the `aria-describedby` attribute and assign it to variable T2.
```
Comment (2biazdk) [inline _rules/SC1-1-1-aria-describedby.md]: "Text Alternative Computation" link leads to a 404.

**Comment by josehilera** [comment]
Some rules include "Repair" property associated to User Question, but the new template for rules does not consider this property:
[https://auto-wcag.github.io/auto-wcag/pages/design/rule-template.html](url)

All rules use the property "Requires context" in User Question tables, but in the rule template the name is only "Context". 
All rules use the property "Requires interaction" in User Question tables, but in the rule template the name is only "Interaction". 
All rules use the property "Presented Item" in User Question tables, but in the rule template the name is "Highlight".

File: _rules/SC1-1-1-text-alternative.md (hunk 18 above)
Comment (josehilera) [inline _rules/SC1-1-1-text-alternative.md]: - Steps 12, 15: Add property Requires interaction: No.
- Steps 4, 6, 7, 17: Requires interaction: No.

File: _rules/SC3-1-1-html.md (hunk 6 above)
Comment (josehilera) [inline _rules/SC3-1-1-html.md]: Step 1: *Do nothing. (This case is covered by [SC3-1-1-xml-lang](SC3-1-1-xml-lang.html).)*
Suggestion: *Execute rule SC3-1-1-xml-lang*

**Pull request merged**